cinema/serializers.py

```diff
@@ -31,9 +31,20 @@ class Meta:
 
 
 class MovieSerializer(serializers.ModelSerializer):
+    image = serializers.ImageField(read_only=True)
+
     class Meta:
         model = Movie
-        fields = ("id", "title", "description", "duration", "genres", "actors")
+        fields = (
+            "id", "title", "description",
+            "duration", "genres", "actors", "image"
+        )
+
+
+class MovieImageSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Movie
+        fields = ("id", "image")
 
 
 class MovieListSerializer(MovieSerializer):
```

Comment:
image must be included here too

Reply:
do I need included it here, if I override class Meta?

Reply:
short answer is yes :+1: